frontend/src/Components/Project/Project.jsx

```diff
@@ -0,0 +1,27 @@
+import "./Project.css"
+
+function Project() {
+    return (
+        <>
+            <tr>
+                <td>Performance</td>
+                <td>In progress</td>
+                <td>Kyle K</td>
+                <td>10/10/2022</td>
+
+            </tr>
+            <tr>
+                <td>Performance</td>
+                <td>In progress</td>
+                <td>Kyle K</td>
+                <td>10/10/2022</td>
+
+            </tr>
+        </>
+
```

Comment:
format

Reply:
Done. I used prettier extension on vscode for formatting and formatted all other files as well.